warehouse/models/gtfs_views/gtfs_schedule_fact_daily_feeds.sql

```diff
@@ -63,6 +68,43 @@ daily_feeds AS (
             AND D.is_in_past_or_present
 ),
 
+daily_service AS (
```

Comment:
**optional/nitpick**: I'm just wondering if we need to alias this table given that there's no join and nothing particularly fancy going on... seems like we could drop the `Services` qualifier on everything. And similarly, I personally prefer using the column name to `group by 1` but that's literally just me, so up to you on whether to change, but wanted to mention that I think in general we do not use column # and have been referencing things by name in group by statements.